server/android/src/android.rs

```diff
@@ -132,7 +133,8 @@ pub mod android {
                     // sync settings need to be configured at runtime
                     sync: None,
                 };
-                let _ = start_server(settings, off_switch_receiver).await;
+                let app_data = AppData::load_from_file().expect("Failed to load app data");
```

Comment:
APP_DATA_FILE, should live in base_dir (can change it from being optional). In android we can only store to certain dirs.

For now would suggest:
* load_from_file and write_to_file to take base_dir as argument
* base_dir to be non optional (you can add to to base.conf and example.conf)